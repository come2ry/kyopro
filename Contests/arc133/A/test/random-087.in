10
11 2 11 9 9 5 11 11 1 7
