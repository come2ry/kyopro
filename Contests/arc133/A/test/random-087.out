1 1 1 1 9 9 5 1 1 1 1 1 7
